ANSWERS.md

```diff
@@ -2,7 +2,25 @@
 
 

```

Comment:
Good Job answering this questions. It is important to be able to verbalize our knowledge. Comes in handy when speaking in interviews.